ddtrace/appsec/iast/_overhead_control_engine.py

```diff
@@ -9,15 +9,24 @@
 
 from ddtrace.appsec.iast._taint_dict import clear_taint_mapping
 from ddtrace.internal.logger import get_logger
+from ddtrace.sampler import RateSampler
 
 
 if TYPE_CHECKING:  # pragma: no cover
     from typing import Set
     from typing import Tuple
     from typing import Type
 
+    from ddtrace.span import Span
+
 log = get_logger(__name__)
 
+
+def get_request_sampling_value():  # type: () -> float
+    # Percentage of requests analyzed by IAST (default: 30%)
+    return float(os.environ.get("DD_IAST_REQUEST_SAMPLING", 30.0))
```

Comment:
Maybe add a try catch in case the variable is set to a non parsable float value like "foo" ?

Reply:
Will do in a follow-up PR